### PR TITLE
Dossantos am patch 1

### PR DESCRIPTION
**Pull request by dossantos-am**

### CODE REVIEW

**Comment by eldu** [comment]
@dossantos-am the assignee for this PR is you, can you add everyone else as reviewers? It's just above the assignee field.


Also, this PR if you take a look at the files added tab, is just an image (amazing by the way!) . Remember to push your changes to the YAML file too !

**Review by eldu — CHANGES_REQUESTED** [review]
Thanks for adding in your about! However, if you look at the preview link (commented in the pr), your entry is not showing up. 

Take a look at some YAML tutorials to better understand the structure. Here's one below! 
https://gettaurus.org/docs/YAMLTutorial/

Also feel free to ask other interns for help! Code buddies is a great place too!

Also please change the title of your PR to something that better describes what this PR does. 

You got this!!!

File: images/people/IMG_8581.jpg
Comment (eldu) [inline images/people/IMG_8581.jpg:1]: Oh actually, i think you need to change this image name to `austin_main.jpg`

**Review by eldu — CHANGES_REQUESTED** [review]
okay... here I am rereviewing. the yaml looks okay i just realized your image should be renamed. Also you'll need 2 images !

File: images/people/IMG_8581.jpg
Comment (dossantos-am) [reply images/people/IMG_8581.jpg:1]: Okay will do, I'll also get the second image in there.

Comment (dossantos-am) [reply images/people/IMG_8581.jpg:1]: I'm not sure what I did after renaming the image to austin_main.jpg but the image is gone.

Comment (eldu) [reply images/people/IMG_8581.jpg:1]: Can you delete and re-add the image?

Comment (eldu) [reply images/people/IMG_8581.jpg:1]: I downloaded it and it doesn't work

**Review by cpaniaguam — COMMENTED** [review]
Seems to be working now 👍

**Review by eldu — APPROVED** [review]
Looks good! 

For future PRs, make sure to name your PRs more specifically what the PR is about.

Also take a look at conventional commits!
https://www.conventionalcommits.org/en/v1.0.0/#summary

Great job!

**Comment by tdivoll** [comment]
Nice work, Austin! First PR right? Feel free to hit the Merge button now that this PR is approved.

**Pull request merged**